packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -330,7 +330,7 @@ Variables used for function parameters must match the function data type. For ex
 | escapeHtml(String html)                                       | Escapes the characters in the provided string using HTML entities                                                             |
 | escapeHtmlAttr(String html)                                   | Encodes data for use in HTML attributes                                                                           |
 
-## Test custom email templates
+## Test your custom email templates
```

Comment:
You might want to search on #test-custom-email-templates .... to see if you broke a cross-reference

Reply:
Right... of course! Will do. Thank you.

Reply:
@brianduffield-okta Turns out it's not an issue. No x-refs!